src/Azure.Functions.Cli/StaticResources/Dockerfile.custom

```diff
@@ -0,0 +1,6 @@
+# To enable ssh & remote debugging on app service change the base image to the one below
+FROM mcr.microsoft.com/azure-functions/dotnet:3.0-appservice 
+ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
+    AzureFunctionsJobHost__Logging__Console__IsEnabled=true
```

Comment:
@ankitkumarr / @divyagandhii  - do you know if it is requrired to set `AzureFunctionsJobHost__Logging__Console__IsEnabled=true` ? when users deploy to azure, this causes duplicated logs in Kusto

Reply:
I would imagine this is required when running or locally or some non App Service environment such as Kubernetes.

Wondering if adding a comment requesting to remove when publishing in Azure would be reasonable.

@ahmelsayed might know better.

Reply:
A separate tracking issue will be opened for this one.